controller/registries/nacos/v2/config.go

```diff
@@ -0,0 +1,207 @@
+// Copyright The HTNN Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v2
+
+import (
+	"fmt"
+	"net/url"
+	"strconv"
+	"sync"
+	"sync/atomic"
+	"time"
+
+	"github.com/nacos-group/nacos-sdk-go/v2/clients"
+	"github.com/nacos-group/nacos-sdk-go/v2/clients/naming_client"
+	"github.com/nacos-group/nacos-sdk-go/v2/common/constant"
+	"github.com/nacos-group/nacos-sdk-go/v2/vo"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"mosn.io/htnn/controller/pkg/registry"
+	"mosn.io/htnn/controller/pkg/registry/log"
+	registrytype "mosn.io/htnn/types/pkg/registry"
+	"mosn.io/htnn/types/registries/nacos"
+)
+
+func init() {
+	registry.AddRegistryFactory(nacos.Name, func(store registry.ServiceEntryStore, om metav1.ObjectMeta) (registry.Registry, error) {
```

Comment:
Since the Nacos v1 & v2 uses the same Nacos registry, we should not register a new registry, but add the v2 support in the existing Nacos implementation.